addons/binding/org.openhab.binding.mcp23017/src/main/java/org/openhab/binding/mcp23017/internal/handler/Mcp23017Handler.java

```diff
@@ -109,12 +109,15 @@ private boolean verifyChannel(ChannelUID channelUID) {
     }
 
     private void handleOutputCommand(ChannelUID channelUID, Command command) {
-        GpioPinDigitalOutput outputPin = pinStateHolder.getOutputPin(channelUID);
-        logger.debug("got output pin {} for channel {} and command {}", outputPin, channelUID, command);
-        if (OnOffType.ON == command) {
-            GPIODataHolder.GPIO.setState(PinState.HIGH, outputPin);
-        } else if (OnOffType.OFF == command) {
-            GPIODataHolder.GPIO.setState(PinState.LOW, outputPin);
+        if (command instanceof OnOffType) {
+            GpioPinDigitalOutput outputPin = pinStateHolder.getOutputPin(channelUID);
+            Configuration configuration = this.getThing().getChannel(channelUID.getId()).getConfiguration();
+
+            // invertLogic is null if not configured
+            boolean activeLowFlag = ACTIVE_LOW_ENABLED.equalsIgnoreCase(configuration.get(ACTIVE_LOW).toString());
```

Comment:
The `active_low` channel configuration parameter needs to be added to [config.xml](https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.mcp23017/ESH-INF/config/config.xml):

Reply:
Can you also use active low input channels @igorarenz?

Reply:
active low for input channels doesnt make sense. Its an feature for output-channels only

Reply:
OK but please add the channel configuration parameter to config.xml :-)

Apparently it works with .thing files, but I don't think users are able to use this channel configuration parameter in UIs such as Paper UI.